leetcode/src/2427.c

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
Please add small documentation explaining the problem.
You can check other files as an example of the same. 🙂

You also need to add this new explanation to the [list of LeetCode problems](https://github.com/TheAlgorithms/C/blob/master/leetcode/README.md).

Reply:
Ok